### PR TITLE
Acc padding

### PR DESCRIPTION
**Pull request by Marr11317**
Resolves: https://trello.com/c/4jEoQVsR/22-add-padding-inside-parentheses-for-bracketed-accidentals

Add padding around bracketed accidentals

New: 
image

Old:
image

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
- [ ] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by RobFog** [comment]
Shouldn't the .gitignore commit be in a separate pull request?

**Comment by Marr11317** [comment]
> 
> 
> Shouldn't the .gitignore commit be in a separate pull request?

I'll remove the commit when the PR is ready.

File: libmscore/sym.cpp [hunk 1]
```diff
@@ -5714,6 +5719,12 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
 
 //    additional symbols
 
+      "Parenthesised double flat accidental",
+      "Parenthesised flat accidental",
+      "Parenthesised natural accidental",
+      "Parenthesised sharp accidental",
+      "Parenthesised double sharp accidental",
```
Comment (Jojo-Schmitz) [inline libmscore/sym.cpp:5726]: I think we'd need them embedded in `QT_TRANSLATE_NOOP("symUserNames", "...")`

Comment (Marr11317) [reply libmscore/sym.cpp:5726]: Do we need them translated though?

Comment (Jojo-Schmitz) [reply libmscore/sym.cpp:5726]: (And the 4 below with a better descriptive string maybe)

Comment (Marr11317) [reply libmscore/sym.cpp:5726]: more generally, is there a rule for which glyph we translate and which ones we don't?

Comment (Jojo-Schmitz) [reply libmscore/sym.cpp:5726]: All that are visible in the UI, like the palettes (except for the symbols palette) or in the status bar

**Comment by Jojo-Schmitz** [comment]
See also https://github.com/w3c/smufl/issues/162

File: libmscore/sym.cpp (hunk 1 above)
Comment (Marr11317) [reply libmscore/sym.cpp:5726]: They shouldn't be exposed to the ui.

except maybed for the master palettes?

**Comment by Marr11317** [comment]
vtest failures are expected

**Comment by Marr11317** [comment]
recommended ligatures are not part of glyphnames.json. see [w3c/smufl#162](https://github.com/w3c/smufl/issues/162) Another fix will be needed for these.

**Comment by vpereverzev** [comment]
A rebase on the latest 3.x should fix these issues with mtests

**Review by vpereverzev — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
> A rebase on the latest 3.x should fix these issues with mtests

So you can as well just merge it

**Comment by Marr11317** [comment]
> 
> 
> > A rebase on the latest 3.x should fix these issues with mtests
> 
> So you can as well just merge it

Just pushed for formality.

**Pull request merged**